packages/bpk-component-phone-input/README.md

```diff
@@ -30,23 +30,23 @@ const getFlag = dialingCode => {
 export default class extends Component {
   constructor(props) {
     super(props);
-    this.state = { dialingCode: '44', phoneNumber: '' }
+    this.state = { dialingCode: '44', phoneNumber: '' };
   }
 
-  onChange = (evt) => {
+  onChange = evt => {
     this.setState({ phoneNumber: evt.target.value });
-  }
+  };
 
-  onDialingCodeChange = (evt) => {
+  onDialingCodeChange = evt => {
     this.setState({ dialingCode: evt.target.value });
-  }
-
+  };
 
   render() {
     return (
       <BpkPhoneInput
         id="phone-input-id"
         name="Telephone input"
+        label="Telephone number"
         placeholder="Telephone number"
```

Comment:
Now that the label is displayed, the placeholder is unnecessary. Could we remove it from the stories, readme, docs and tests?